packages/parser/package.json

```diff
@@ -11,6 +11,10 @@
       "require": "./dist/index.cjs",
       "import": "./dist/index.js"
     },
+    "./lib/assertions": {
```

Comment:
Should this be considered a breaking change or is it safe to treat as a minor release?